run_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
I would suggest removing this file and instead adding `--no-cov` when needed to `pytest`.